packages/visualizations/addon/models/metric-label.js

```diff
@@ -43,7 +43,7 @@ export default VisualizationBase.extend(Validations, {
     let metrics = Ember.A( get(request, 'metrics') ),
         metric =  get(metrics, 'firstObject.canonicalName'),
         description =  metricFormat(get(metrics, 'firstObject'), get(metrics, 'firstObject.metric.longName')),
-        allFormats = get(MetricLabelConfig.proto(), 'predefinedFormats'),
+        allFormats = get(NumberFormatSelector.proto(), 'predefinedFormats'),
```

Comment:
This seems a little weird that we need the component class to get the `predefinedFormats`. Can we think of another way to do this? Maybe use an [enum](https://github.com/yahoo/navi/tree/master/packages/reports/addon/utils/enums)?